src/domain/messages/helpers/message-verifier.helper.ts

```diff
@@ -141,17 +141,9 @@ export class MessageVerifierHelper {
       signature: args.signature,
     });
 
-    if (
-      !this.isEthSignEnabled &&
-      signature.signatureType === SignatureType.EthSign
-    ) {
-      throw new HttpExceptionNoLog(
-        'eth_sign is disabled',
-        MessageVerifierHelper.StatusCode,
-      );
-    }
-
-    const isBlocked = this.blocklist.includes(signature.owner);
+    const isBlocked = this.blocklist.some((blockedAddress) => {
+      return isAddressEqual(signature.owner, blockedAddress);
```

Comment:
I think it was because of mock data but I came across some inconsistencies with non-checksummed addresses. As such, I migrated to using this helper in both verifiers to be safe.